src/LinkDotNet.Blog.Web/Controller/RssFeedController.cs

```diff
@@ -34,18 +35,28 @@ public RssFeedController(
 
         description = introductionConfiguration.Value.Description;
         blogName = applicationConfiguration.Value.BlogName;
+        blogPostsPerPage = applicationConfiguration.Value.BlogPostsPerPage;
         this.blogPostRepository = blogPostRepository;
     }
 
     [ResponseCache(Duration = 1200)]
     [HttpGet]
-    public async Task<IActionResult> GetRssFeed()
+    public async Task<IActionResult> GetRssFeed([FromQuery] bool withContent = false, [FromQuery] int? numberOfBlogPosts = null)
     {
+        if (!ModelState.IsValid)
```

Comment:
Out of curiosity: How can we come into that bid?

Reply:
The compiler would cry if I didn't have that, and since warnings are treated as errors, I added it :laughing:

Reply:
Interesting! Leave it in then :D